articles/integrations/aws/index.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
And I would simplify the title to `Integrate Auth0 with Amazon Web Services (AWS)`